docs/geedocs/docsrc/answer/geeServerConfigAndSecurity/postgreSqlAuthentication.rst

```diff
@@ -47,12 +47,12 @@ PostgreSQL authentication
       -  The ``mod_authn_dbd`` module provides the authentication
          back-end with a PostgreSQL server. For more information about
          the module, see the
-         `mod_authn_dbd <http://www.google.com/url?q=http%3A%2F%2Fhttpd.apache.org%2Fdocs%2F2.2%2Fmod%2Fmod_authn_dbd.html&sa=D&sntz=1&usg=AFrqEzcLzX90MM12j49RNuWEy_X7y6wTrQ>`__
+         `mod_authn_dbd <http://www.google.com/url?q=http%3A%2F%2Fhttpd.apache.org%2Fdocs%2F2.4%2Fmod%2Fmod_authn_dbd.html&sa=D&sntz=1&usg=AFrqEzcLzX90MM12j49RNuWEy_X7y6wTrQ>`__
```

Comment:
Can we update the url to point directly to: https://httpd.apache.org/docs/2.4/mod/mod_authn_dbd.html ?